src/table/TableBodyWrapper.jsx

```diff
@@ -68,56 +72,96 @@ const TableBodyWrapper = ({
     addSelectionListeners(selectionsAPI, selDispatch, setShouldRefocus);
   }, []);
 
+  useEffect(() => {
+    setSrNotation('');
+
+    setTimeout(() => {
+      setSrNotation(getSrNotation());
+    }, MAX_NOTATION_DELAY);
+  }, [focusedCellCoord, selState]);
+
+  const getSrNotation = () => {
+    if (focusedCellCoord.toString() == '0,0') return '';
+
+    const [x, y] = focusedCellCoord;
```

Comment:
fyi these are not x, y, if anything y, x. Or just rowIdx, colIdx for clarity. I think I put them in that order since that is the way the dom is ordered

Reply:
✅ Fixed